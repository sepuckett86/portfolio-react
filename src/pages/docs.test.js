import React from 'react';
import { shallow } from 'enzyme';
import Docs from './docs';

describe('Docs page', () => {
  it('renders', () => {
    const wrapper = shallow(<Docs />);
    expect(wrapper).toMatchSnapshot();
  });
});
