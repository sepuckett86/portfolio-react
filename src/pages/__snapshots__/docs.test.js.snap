// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Docs page renders 1`] = `
<Fragment>
  <DocsContainer />
</Fragment>
`;
